tests/ui/rc_buffer.rs

```diff
@@ -0,0 +1,13 @@
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::rc::Rc;
+
+#[warn(clippy::rc_buffer)]
+struct S {
```

Comment:
Lets add some negative cases as well. Off the top of my head I think of `Rc<RefCell<String>>` or possibly even `struct String;` followed by `Rc<String>`

Also, tests for function signatures and not just struct definitions.

Reply:
Ok, will do.